team-telnyx/telnyx-webrtc-android
# [WEBRTC-535] Add a way to cancel outgoing call

Signed-off-by: Lucas Pais <lucasp@7g804d3.office.telnyx.com>

[WebRTC-535 - Ticket Description.](https://telnyx.atlassian.net/browse/WEBRTC-535)

---
Add a way to cancel an outgoing invitation, before an Answer is received.

## :older_man: :baby: Behaviors
### Before changes
We didn't have a way to cancel an outgoing call, nor to receive feedback on whether an invitation is in progress.

### After changes
Green call button, will turn red indicating an invitation is in progress and we can cancel it.

## ✋ Manual testing
1. Call
2. Press call button again (now will be RED)
3. Verify call is no longer in progress, and ring has stopped on the receiver.
